packages/analytics-js/__tests__/components/eventRepository/EventRepository.test.ts

```diff
@@ -354,29 +354,32 @@ describe('EventRepository', () => {
     );
   });
 
-  it('should buffer the data plane events if the pre-consent event delivery strategy is set to buffer', () => {
-    const eventRepository = new EventRepository(
-      mockPluginsManager,
-      defaultStoreManager,
-      defaultHttpClient,
-      defaultErrorHandler,
-      defaultLogger,
-    );
+  it.each(['session', 'anonymousId'] as const)(
```

Comment:
Let's include `none` also in this array.

Reply:
@saikumarrs done. I amended my previous commit with the suggested change.